CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.26.0 Release
+- Release Date: December 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
+
+#### ID-loaded push notifications (E20)
+
+#### Upgraded RN to v0.61
+
+### Improvements
+- Added support for pasting other file types such as videos, PDFs, and Google Docs.
+- Added the option to convert public channels to private in the channel info screen.
+- Made usernames in system messages tappable.
+- Added a count for pinned posts header icon.
+- Added an autocomplete to edit post screen.
+- Updated the channel name length character limit to 64 to match server.
+- Added an expand button to truncated markdown tables to improve discoverability of opening them in full screen.
+- Added an error message when trying to share too long text from share extension.
+- Improved behaviour where posts from different authors in the same thread appeared to be from different threads if separated by new message line.
+- Added support for native emojis in the emoji picker and autocomplete.
+- Removed "Add Reaction" option from the long post view.
+- Improved behaviour of large number of emojis when a post changed from horizontal scroll to wrap.
+- Added support for a generic error message in interactive dialog responses.
+- Added the ability to disable attachment buttons and fields.
+
```

Comment:
channel drawer button now has an accessibility label that is read with voice over